net/frr/src/opnsense/service/templates/OPNsense/Quagga/bgpd.conf

```diff
@@ -155,15 +155,17 @@ router bgp {{ OPNsense.quagga.bgp.asnumber }}
 {%     endif %}
 {%   endfor %}
  exit-address-family
-!
+ !
 {%  endfor %}
-
+!
 {%   if helpers.exists('OPNsense.quagga.bgp.prefixlists.prefixlist') %}
 {%     for prefixlist in helpers.sortDictList(OPNsense.quagga.bgp.prefixlists.prefixlist, 'name', 'seqnumber' ) %}
 {%       if prefixlist.enabled == '1' and prefixlist.version == 'IPv4' %}
 ip prefix-list {{ prefixlist.name }} seq {{ prefixlist.seqnumber }} {{ prefixlist.action }} {{ prefixlist.network }}
 {%       endif %}
+{%     endfor %}
 !
+{%     for prefixlist in helpers.sortDictList(OPNsense.quagga.bgp.prefixlists.prefixlist, 'name', 'seqnumber' ) %}
```

Comment:
Is there a functional reason to traverse the list twice?

Reply:
I am running over the list twice so I get distinct sections for IPv4 and IPv6 Prefixes in the config. This mimics the generation done for example by the pfSense bgp config generator.
The output looks something like this:
![image](https://github.com/opnsense/plugins/assets/127999166/fd74d423-f0ba-44ca-856e-43a98c4b4b48)

Reply:
ok, so this doesn't change anything functional if I understand you correctly. But when we don't sort them, will `vtysh` keep our order when querying or does it regroup as well?